tests/apollo/test_skvbc_preexecution.py

```diff
@@ -191,3 +193,32 @@ async def test_view_change(self, bft_network):
                 break
 
         self.assertEqual(new_last_block, last_block)
+
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    async def test_f_nonprimary_crash_amid_parallel_tx(self, bft_network):
+        '''
+        Crash f nonprimary replicas and submit X parallel write submissions.
+        Block processing of the network should be unaffected with f-count interruption.
+        Final block length should match submitted transactions count exactly.
+        '''
+        bft_network.start_all_replicas()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        
+        num_of_requests = 10
+        clients = bft_network.random_clients(1)
```

Comment:
do you still need the `clients` variable?

Reply:
I will just use only one client for the whole test, gotten from `bft_network.random_client()`.